vmc/resource_vmc_sddc_test.go

```diff
@@ -139,6 +139,10 @@ resource "vmc_sddc" "sddc_1" {
     account_link_sddc_config {
     customer_subnet_ids  = [data.vmc_customer_subnets.my_subnets.ids[0]]
     connected_account_id = data.vmc_connected_accounts.my_accounts.id
+	}
```

Comment:
formatting seems off between lines 142 and 145

Reply:
I am not able to see the formatting as off in my IDE. I only see it here on Github.